src/template.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Leaderboard</title>
+</head>
+<body>
+  <h1 class="page-header">Leaderboard</h1>
+  <div class="container">
+    <div class="scores-container">
+      <div class="scores-head-container">
+        <h2 class="recent-scores-header">Recent Scores</h2>
+        <button type="button" id="refreshBtn">Refresh</button>
+      </div>
+      <div id="scoresList">
+      </div>
+    </div>
+    <form class="score-form">
+      <h3 class="form-header">Add Your Score</h3>
+      <input type="text" placeholder="Your Name" id="input">
+      <input type="text" placeholder="Your Score" id="input">
+      <button id="formBtn" type="submit">Submit</button>
+    </form>
+  </div>
```

Comment:
- [ ] Great job working on the set up so far but I think it would be great if you made the UI as the given template since your design is different so please as required change the layout to have the same as given layout no styling at this time 💪🏻🤩💯

![image](https://user-images.githubusercontent.com/58553711/231330248-889fd61e-c2ad-435d-a933-706dd2864f21.png)

- Your layout 💯
![image](https://user-images.githubusercontent.com/58553711/231330336-6508b6fd-b61e-4f99-ae6c-8429c2c2fcfa.png)

Reply:
Hi @Reem-lab, Thank you for your feedback. Kindly run the command ```npm run dev``` as this the script used for it as mentioned in the readme.md file, and then you should find my UI looks like this 
![image](https://user-images.githubusercontent.com/46492634/231336050-35235fa7-eba1-4a14-8ec0-8516564846de.png)